src/common-utils/scripts/bash_theme_snippet.sh

```diff
@@ -23,3 +22,20 @@ __bash_prompt() {
 }
 __bash_prompt
 export PROMPT_DIRTRIM=4
```

Comment:
Can you help fix the test failures? thanks!

Reply:
@osortega Can you check why https://github.com/osortega/features/blob/0556f86d4db352c75888c04cd303200bc5cc44f3/test/common-utils/configure_zsh_as_default_shell.sh#L12 check started failing with your PR changes? 

Now the test is green but I think that's because of the `TERM` condition.

Reply:
This is fixed